src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius) {
```

Comment:
you should pass color in the constructor and initialize it using Figure constructor, the same with other figures